wordpress-mobile/WordPress-Android
# Use reader endpoint for menu item translations

Fixes #5748. Now that the endpoint has been updated we can get our menu item (Discover, Followed Tags) translations from it.

To test:
* Go to Reader and check that the menu items are in your app's language
* Change the App language via Me->App Settings
* Verify the Reader menu items are in the newly selected language
